homeassistant/components/barry/sensor.py

```diff
@@ -0,0 +1,63 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(hours=1)
+MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Barry sensor."""
+    barry_connection = hass.data.get(DOMAIN)
+    price_code = hass.data.get("price_code")
+    async_add_entities([BarrySensor(barry_connection, price_code)], True)
+
+
+class BarrySensor(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, barry_home, price_code):
+        """Initialize the sensor."""
+        self._barry_home = barry_home
+        self.price_code = price_code
+        self._state = None
+        self._name = "Barry"
```

Comment:
```suggestion
        self._attr_name = "Electricity price Barry"
        self._attr_unit_of_measurement = "DKK/kWh"
```

Reply:
Thanks for the suggetion. I made the changes as per the requested but by using "_attr_name" and "_attr_unit_of_measurement". I need to set "unit_of_measurement" and "name" property instead to make ir display besides the states column.

Reply:
You can remove the property methods and replace them with the class attributes. See:

https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes

It reduces the amount of logic needed